ProcessList.c

```diff
@@ -624,8 +626,6 @@ void ProcessList_scan(ProcessList* this, bool pauseProcessUpdate) {
             // immediately remove
             ProcessList_remove(this, p);
          }
-      } else {
-         p->updated = false;
```

Comment:
Why is this no longer needed?
I do not see where else it might be set, so the previous condition `} else if (p->updated == false) {` can trigger on no-longer existing entries.

Reply:
This is set before each run of `Process_goThroughProcesses` as an initialization step.
https://github.com/BenBE/htop/blob/command_color_instr/ProcessList.c#L589

Reply:
true